modules/openapi-generator-gradle-plugin/build.gradle

```diff
@@ -90,6 +96,23 @@ gradlePlugin {
     }
 }
 
+pluginBundle {
+    // These settings are set for the whole plugin bundle
+    website = 'https://openapi-generator.tech/'
+    vcsUrl = 'https://github.com/OpenAPITools/openapi-generator'
+    description = 'OpenAPI Generator allows generation of API client libraries (SDK generation), server stubs, documentation and configuration automatically given an OpenAPI Spec (v2, v3)'
+
+    plugins {
+        // first plugin
+        openApiGenerator {
+            id = 'org.openapi.generator'
+            displayName = 'OpenAPI Generator Gradle Plugin'
+            tags = ['openapi-3.0', 'openapi-2.0', 'openapi', 'swagger', 'codegen', 'sdk']
```

Comment:
I wasn't sure what tags to use here.

Reply:
I think those are good to start with and we always revise and add more later based on the feedback from the community.